README.md

```diff
@@ -31,13 +31,27 @@ targets: '[{"project": "Backlog", "section": "Development Done"}, {"project": "C
 if you don't want to move task omit `targets`.
 
 
-## Example usage
+## Examples
+
+
+#### Without special chracters:
 
 ```yaml
-uses: https://github.com/insurify/github-actions@v2.0.0
+uses: insurify/github-asana-action@v3.0.0
 with:
   asana-pat: 'Your PAT'
   task-comment: 'View Pull Request Here: '
   trigger-phrase: 'Asana Task:'
   targets: '[{"project": "Backlog", "section": "Development Done"}, {"project": "Current Sprint", "section": "In Review"}]'
+```
+
+#### With special chracters:
+
+```yaml
+uses: insurify/github-asana-action@v3.0.0
+with:
+  asana-pat: 'Your PAT'
+  task-comment: 'View Pull Request Here: '
+  trigger-phrase: "\\*\\*Asana Task:\\*\\*"
```

Comment:
We can use [escape-string-regexp](https://www.npmjs.com/package/escape-string-regexp) or main [function](https://github.com/sindresorhus/escape-string-regexp/blob/master/index.js#L3) from it to convert any string to valid escaped regular expression string, so end-user don't need to go into the hassle of writing a valid regular expression.